packages/theme-chalk/src/button.scss

```diff
@@ -225,4 +232,30 @@ $button-icon-span-gap: map.merge(
       }
     }
   }
+
+  @include m(link) {
+    border-color: transparent;
+    color: getCssVar('color', 'primary');
+    background: transparent;
+    padding-left: 0;
+    padding-right: 0;
+
+    &:hover,
+    &:focus {
+      color: getCssVar('color', 'primary', 'light-3');
+      border-color: transparent;
+      background-color: transparent;
```

Comment:
Is `border-color: transparent`、`background-color: transparent` necessary? I see lines 237 and 239 have been added.

Reply:
This is just adding back the styles I removed.

Reply:
Yes, just did a restore.